README.md

```diff
@@ -77,6 +77,44 @@ const response = await firestoreService.get('regions/32');
 
 The response will have the information about the region with id 32.
 
+#### Tools to make queries
+
+Firestore service allows you to use certain tools to manipulate your data in the query. For this you should add a body into the request.
+
+- Limit
+
+The response will only have 20 "regions"
+
+```js
+const response = await firestoreService.get('regions/32', { limit: 20});
+```
+
+- Filter
+
+The response will only have users who are older than 22 years old and younger than 35 
+
+```js
+const response = await firestoreService.get(
```

Comment:
this has the same problem, I'd ask someone from backend how to send the filters here using query params.